### ISSUE TITLE
Have created variables and psuedo coded function, need assistance implementing. No errors because i have not created code to get an error.

### ISSUE THREAD

**Issue by majoseph77** [majoseph77/mancala_game_project1#3]
# User Story
- As a user I want to be able to know how many stones are in each of my pits
  and how many stones are in my opponents pits so I can know which option would
  be the best move.
## step1 write in plain english:
- I want to take the value of any pit and add that value to the following pits by 1 unit until the original amount is 0. Then take the amount of stones in the pit that was last added to by the first pit anc continue the same process of adding to sequential pits by 1 until the last stone is dropped in a pit with the value of 0. Then it will be the next players turn
## step 2 write a fx signature:
- name of fx : move
- inputs: "numstones", "empty", "player1Side", "player2Side"
- returns: new "numstones"
- side effects: "updateDisplay"
  `function move = function (numstones, empty, player1Side, player2Side){
  };`
## step3 implement function:

`function move = function (numstones, empty, player1Side, player2Side){
step1: get value of pit that has been clicked: value = 4
step 2: value grabbed is 4 add 1 stone clockwise to the next 4 pits. 
// the next 4 pits have a starting value of 4, so the new value of the next 4 pits is 5
//
step 3: Take the value of the last pit that the stone was added to and continue the process of adding one to the next 5 pits until the last stone is dropped in a pit with a value of 0.
step 4: display that it is the next players turn.
};

**Comment by ghost** [comment]
Good issue, Meghan! (Almost great... :smile:)

However, two things:
1. I don't know the problem. You look on the right track!
2. When adding blocks of code, don't use single backticks, use triple backticks, or code fences, eg:

<pre>
```javascript
// code here can be on
// multiple lines...
```
</pre>
